server/src/user/test/user.e2e-spec.ts

```diff
@@ -88,18 +88,14 @@ describe('UserController (e2e)', () => {
     });
   });
 
-  describe('DELETE /me/?house_id=:houseId', () => {
-    const houseId = 106;
-
+  describe('DELETE /me/', () => {
```

Comment:
You can just do `/me` and other parts too!

Reply:
I fixed it.
db23bcabe05a9713df074ffcd7e9ed0ecb1cbed3